spec/api-spec.json

```diff
@@ -1,35 +1,35 @@
 {
   "openapi": "3.0.2",
   "info": {
-    "title": "sic-Code-api",
+    "title": "sic-code-api",
     "version": "1.0"
   },
   "servers": [
     {
-      "url": "http://api.chs.local/sic-code-search",
+      "url": "http://api.chs.local/internal",
       "description": "Development server"
     }
   ],
   "paths": {
-    "/search": {
+    "/sic-code-search": {
       "post": {
-        "summary": "Search for SIC code via economic activity",
+        "summary": "Search for SIC code via economic-activity/sic-description/sic-code",
```

Comment:
remove the hyphens here (since they are not used in other parts of this file for these fields)